From 9966f0a24e7d62e2006277bbb542eff8c6b7a356 Mon Sep 17 00:00:00 2001
From: Kovid Goyal <kovid@kovidgoyal.net>
Date: Thu, 13 Aug 2020 22:47:51 +0530
Subject: [PATCH 385/408] Fix next_screen_location()

The change to column_at() to consider points on the boundary as
belonging to the left column broke next_screen_location()
---
 src/pyj/read_book/paged_mode.pyj | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/pyj/read_book/paged_mode.pyj b/src/pyj/read_book/paged_mode.pyj
index b970df5ca6..01215e061d 100644
--- a/src/pyj/read_book/paged_mode.pyj
+++ b/src/pyj/read_book/paged_mode.pyj
@@ -377,7 +377,7 @@ def next_screen_location():
     if is_full_screen_layout:
         return -1
     cc = current_column_location()
-    ans = cc + screen_inline
+    ans = cc + screen_inline + 1
     if cols_per_screen > 1 and 0 < number_of_cols_left() < cols_per_screen:
         return -1  # Only blank, dummy pages left
     limit = scroll_viewport.paged_content_inline_size() - scroll_viewport.inline_size()
-- 
2.28.0

